### PR TITLE
Add randommm.py file

### PR DESCRIPTION
**Pull request by BlueSkies503**
I began work on random number generating script.

### CODE REVIEW

File: randommm.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# this also works
+from random import randint
+num = randint(1, 10)
+print(num)
```
Comment (dodecaphonix) [inline randommm.py]: No newline at end of file

**Comment by dodecaphonix** [comment]
The branch says you are adding a route. You should add it.

**Comment by BlueSkies503** [comment]
Thank you for your careful attention to detail.

File: randommm.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from random import randint
```
Comment (dodecaphonix) [inline randommm.py]: We can probably delete this file for now.

File: routes.py [hunk 3]
```diff
@@ -8,5 +10,10 @@ def hello_world():
     return "<h1>Hello, World!</h1>"
 
 
+@app.route("/foo")
+def rand_element():
+    return f"<h1>{randint(1, 20)}</h1>"
```
Comment (dodecaphonix) [inline routes.py]: A couple of things with this line.

1: The 1 and the 20 are called magic numbers. Picture reading a file with a thousand lines and then a random unexplained integer in the middle. What is it? Is it safe to change it? Who knows. They should be defined as constants with descriptive names in UPPER_SNAKE_CASE at the top of this file or in a dedicated constants.py file and imported here.

2: We should probably update this to return a rendered template like current master is doing. You have options for getting you branch up to date with master. You can rebase you branch of latest master locally and force push to your remote branch or you can merge master into your branch.

File: routes.py [hunk 4]
```diff
@@ -8,5 +10,10 @@ def hello_world():
     return "<h1>Hello, World!</h1>"
 
 
+@app.route("/foo")
```
Comment (dodecaphonix) [inline routes.py]: Update to a more descriptive path.

File: randommm.py (hunk 2 above)
Comment (BlueSkies503) [reply randommm.py]: Should I delete it here in github?

File: routes.py (hunk 4 above)
Comment (BlueSkies503) [reply routes.py]: How about `/scramble` cuz this is where the scrambling algorithm will eventually output?

Comment (dodecaphonix) [reply routes.py]: I'd name it what it's currently doing. Let's keep the path in line with the functionality. We can always rename it or add another route later.

File: randommm.py (hunk 2 above)
Comment (dodecaphonix) [reply randommm.py]: You either need to:
- delete it here and then fetch+rebase locally
- delete it locally and push the change

They are totally equivalent, so do whichever you prefer.

I always get things how I want them locally and then push. I much prefer the terminal to any UI abstractions and I like to make sure everything is perfect before I make it publicly visible.

Comment (BlueSkies503) [reply randommm.py]: I think I agree with you on that.

File: routes.py (hunk 4 above)
Comment (BlueSkies503) [reply routes.py]: Good thinking, thanks.

File: routes.py (hunk 3 above)
Comment (dodecaphonix) [reply routes.py]: How do we un-outdate this comment? Will adding this comment do it?

Comment (dodecaphonix) [reply routes.py]: I see. The associated line no longer exists. Let me know what you want to talk about how to do 2.

File: randommm.py (hunk 2 above)
Comment (dodecaphonix) [reply randommm.py]: Feel free to click the "Resolve conversation" button for this thread.

File: routes.py [hunk 5]
```diff
@@ -8,5 +15,10 @@ def hello_world():
     return "<h1>Hello, World!</h1>"
 
 
+@app.route("/rand_num")
+def rand_element():
```
Comment (dodecaphonix) [inline routes.py]: No reason not to make this function name match the path.

File: routes.py (hunk 4 above)
Comment (dodecaphonix) [reply routes.py]: Feel free to click the "Resolve conversation" button for this thread.

File: randommm.py (hunk 1 above)
Comment (dodecaphonix) [reply randommm.py]: Feel free to click the "Resolve conversation" button for this thread.

File: routes.py (hunk 3 above)
Comment (BlueSkies503) [reply routes.py]: Definitely going to need help with number 2. I'm totally lost in "parallel universes" of git right now.

**Review by dodecaphonix — APPROVED** [review]
LGTM

**Pull request merged**